### PR TITLE
Add ASG cooldown and health_check_grace_period

### PR DESCRIPTION
**Pull request by ArieLevs**
# PR o'clock

## Description

Add [default_cooldown](https://www.terraform.io/docs/providers/aws/r/autoscaling_group.html#default_cooldown)
and [health_check_grace_period](https://www.terraform.io/docs/providers/aws/r/autoscaling_group.html#health_check_grace_period)
options

### Checklist

- [x] Change added to CHANGELOG.md. All changes must be added and breaking changes and highlighted
- [x] CI tests are passing
- [x] README.md has been updated after any changes to variables and outputs. See https://github.com/terraform-aws-modules/terraform-aws-eks/#doc-generation

### CODE REVIEW

**Comment by ArieLevs** [comment]
@max-rocket-internet 
@barryib 
Opened new PR

**Comment by ArieLevs** [comment]
btw, while fixing my pr, i've noticed this https://github.com/terraform-aws-modules/terraform-aws-eks/blob/master/workers.tf#L77
shouldn't it be `var.worker_groups[count.index],` as well?

**Comment by barryib** [comment]
> btw, while fixing my pr, i've noticed this https://github.com/terraform-aws-modules/terraform-aws-eks/blob/master/workers.tf#L77
> shouldn't it be `var.worker_groups[count.index],` as well?

Thanks @ArieLevs I was a bit tired yesternight. Fixing this.

**Comment by barryib** [comment]
For the record, supersedes https://github.com/terraform-aws-modules/terraform-aws-eks/pull/751

File: CHANGELOG.md [hunk 1]
```diff
@@ -13,6 +13,8 @@ project adheres to [Semantic Versioning](http://semver.org/).
 - Adding `encrypted` option to worker's root_block_device as read from the worker configurations (by @craig-rueda)
 - Use correct policy arns for CN regions (cn-north-1, cn-northwest-1) (by @cofyc)
 - Add support for ASG max instance lifetime (by @sidprak)
+- Add `default_cooldown` option to workers ASG (by @ArieLevs)
+- Add `health_check_grace_period` option to workers ASG (by @ArieLevs)
```
Comment (barryib) [inline CHANGELOG.md]:
```suggestion
- Add `default_cooldown` and `health_check_grace_period` options to workers ASG (by @ArieLevs)
```

**Comment by barryib** [comment]
Thanks @ArieLevs

**Pull request merged**